dht/go.mod

```diff
@@ -1,38 +1,122 @@
 module github.com/libp2p/test-plans/dht
 
-go 1.14
+go 1.19
```

Comment:
Since we're bumping the go version here, are you sure the test is passing without bumping the docker builder's version aswell?

I'd expect something like:

```toml
[global.build_config]
      build_base_image = 'golang:1.19-buster'
```

Reply:
(resolving, it was updated in the manifest 👍 )

Reply:
yes, I added that to `manifest.toml`:
```
[builders."docker:go"]
enabled = true
go_version = "1.19"
build_base_image = "golang:1.19-buster"
module_path = "github.com/libp2p/test-plans/dht"
exec_pkg = "."
```

 It appears to work as-is, should I add the `build_base_image` line to the composition files as well?